pynest/examples/twoneurons.py

```diff
@@ -22,32 +22,61 @@
 """Two neuron example
 ----------------------------
 
+This script simulates two connected pre- and postsynaptic neurons.
+The presynaptic neuron receives a constant external current.
+And, the membrane potential of both neurons are recorded.
 
 See Also
-~~~~~~~~~~
+~~~~~~~~
 
 :doc:`one_neuron`
 
 """
 
+###############################################################################
+# First, we import all necessary modules for simulation, analysis and plotting.
 
 import nest
 import nest.voltage_trace
 import matplotlib.pyplot as plt
 
+nest.set_verbosity("M_WARNING")
+nest.ResetKernel()
```

Comment:
Should probably add something about these to not confuse new users.

Reply:
thanks, just added a description about that